src/foundry/common/primitives/math.d.mts

```diff
@@ -43,11 +43,12 @@ declare global {
     roundDecimals(number: number, places: number): number;
 
     /**
-     * Round a floating-point number using the fastest available method.
-     * This should be used instead of Math.round in cases where performance matters.
-     * A key limitation is this method returns zero if the input is NaN or undefined.
+     * To keep compatibility with previous implementation.
+     * roundFast was bugged and the performance advantage was not there.
      * @param number - A finite number
      * @returns The rounded number
+     *
+     * @deprecated since v10 in favor of `Math.round`
```

Comment:
I've been doing `since v10, will be removed in v12` for this then stuffing the deprecation message in the next line with `
```ts
// @remarks `"roundFast is deprecated in favor of Math.round"`
```

Reply:
The source actually doesn't specify when it will be removed, but I can move the "in favor of" to remarks if you prefer.

Reply:
That's what the line `foundry.utils.logCompatibilityWarning(msg, {since: 10, until: 12});` does

Reply:
Oh ok, makes sense. I'll make the change to keep things consistent.